src/config/entities/configuration.ts

```diff
@@ -45,7 +46,7 @@ export default () => ({
     isProduction: process.env.CGW_ENV === 'production',
     // Enables/disables the execution of migrations on startup.
     // Defaults to true.
-    runMigrations: process.env.RUN_MIGRATIONS?.toLowerCase() !== 'false',
+    runMigrations: process.env.RUN_MIGRATIONS?.toLowerCase() === 'false',
```

Comment:
This is changing the original behavior (and making the comment incosistent), as the default value is no longer `true`.

It should be `!== false` instead of `=== false`, as it is on the `main` branch.

Reply:
I saw you already changed this, thanks!

Reply:
Was about to write. This was a mistake. Fixed here:
https://github.com/safe-global/safe-client-gateway/pull/1988/commits/378fdd1d0c485f4b3dbdba8a8ed0efc33d8a2121